doc/src/records/dhwheater-doc.md

```diff
@@ -225,7 +225,9 @@ Instanteous water heater load carry forward factor -- approximate number of hour
 
 **whZone=*znName***
 
-Name of zone where water heater is located, used only in detailed HPWH models (whHeatSrc=ASHPX or whHeatSrc=RESISTANCEX), otherwise no effect. Zone conditions are used for tank heat loss calculations.  Heat exchanged with the DHWHEATER are included in the zone heat balance.  whZone also provides the default for whASHPSrcZn (see below). whZone and whTEx cannot both be specified.
+Name of zone where water heater is located, used only in detailed HPWH models (whHeatSrc=ASHPX or whHeatSrc=RESISTANCEX), otherwise no effect. Zone conditions are used for tank heat loss calculations.  Heat losses from the DHWHEATER are included in the zone heat balance.  whZone also provides the default for whASHPSrcZn (see below).
+
+whZone and whTEx cannot both be specified.
```

Comment:
These changes don't seem related. Just want to confirm that it is intentional to make them on this branch.